CONTRIBUTING.md

```diff
@@ -62,7 +62,7 @@ If your application has dependencies other than `@mui/toolpad`, you have to temp
 1. Make sure to start the build in watch mode again and the run the app with
 
    ```bash
-   pnpm toolpad dev examples/qr-generator --dev
+   pnpm toolpad dev examples/qr-generator/toolpad --dev
```

Comment:
If I don't include `toolpad` it creates the files outside the `toolpad` folder, that's not expected, right? Or am I doing something wrong?

Reply:
It works well after this change and it matches the command that was already in step 3 above it, so I'm pretty sure this should be the way to write it.